src/test/resources/META-INF/frontend/styles/fc-badge-list.css

```
@@ -1,4 +1,23 @@
-/* FOR VAADIN 25 */
+/*-
```

Stylelint (17.7.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)